src/server/oauth/providers/oauth.ts

```diff
@@ -270,7 +26,10 @@ export type OAuthConfigInternal<Profile> = Omit<
     url: URL;
     request?: TokenEndpointHandler["request"];
     clientPrivateKey?: CryptoKey | PrivateKey;
-    /** @internal */
+    /**
+     * @internal
+     * @deprecated
```

Comment:
we probably shouldn't be adding a `@deprecated`? Potentially from a bad rebase?

Reply:
Oh woops, missed this comment. I'll address in a follow-up.